cdk/lib/__snapshots__/manage-frontend.test.ts.snap

```diff
@@ -2359,6 +2399,9 @@ Object {
             "Ref": "AMIManagefrontend",
           },
           "InstanceType": "t4g.small",
+          "MetadataOptions": Object {
+            "InstanceMetadataTags": "enabled",
+          },
```

Comment:
this is to allow access to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/work-with-tags-in-IMDS.html
This is probably useful for kibana logs or wazuh or something but shouldn't break anything.